content/comments-in-swift.md

```diff
@@ -0,0 +1,44 @@
+<!--
+---
+Subjects: # Please first look to use Subjects that are already listed at author.codecademy.com/browse/categories
+  - "Learn Swift"
+  - "iOS and Swift Fundamentals"
+Tags: # Please first look to use tags that are already listed at author.codecademy.com/browse/tags
```

Comment:
No need to include the `#`s! I'll specify that in the template, since it's a totally understandable thing to do.